packages/suite/src/views/wallet/trading/common/TradingUtils/TradingUtilsPrice.tsx

```diff
@@ -43,12 +44,19 @@ export const TradingUtilsPrice = ({
     receiveAmount,
     receiveCurrency,
 }: TradingCryptoAmountProps) => {
-    const { type } = useTradingFormContext();
+    const { type, quotesRequest } = useTradingFormContext();
+
+    const showProviderAdjustedAmountTooltip =
+        type === 'sell' &&
```

Comment:
Sorry, I like `isTradingSellContext` so much. :-D
Nit:

```suggestion
    const context = useTradingFormContext();
    const { type, quotesRequest } = context;

    const showProviderAdjustedAmountTooltip =
        isTradingSellContext(context) &&
```

Reply:
done